source/agora/consensus/data/UTXO.d

```diff
@@ -53,6 +53,133 @@ public struct UTXO
     }
 }
 
+/*******************************************************************************
+
+    This is the the base class of a UTXO storage.
+    It is currently inherited by UTXOSet and TestUTXOSet.
+
+*******************************************************************************/
+
+abstract class UTXOCache
+{
+    /// Keeps track of spent outputs during the validation of a Tx / Block
+    protected Set!Hash used_utxos;
+
+    /***************************************************************************
+
+        Add all of a transaction's outputs to the Utxo set,
+        and remove the spent outputs in the transaction from the set.
+
+        Params:
+            tx = the transaction
+            height = Height of the block where UTXO will be unlocked
```

Comment:
Isn't this the wrong documentation? Because in the body of the function `+1` is used. Therefore `height` is just the current block height is it not?

Reply:
Good catch, it's the current height.